src/js/angular/ttyg/templates/chat-panel.html

```diff
@@ -1,14 +1,10 @@
 <link href="css/ttyg/chat-panel.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
 
-<div class="chat-panel h-100">
+<div class="chat-panel h-100" ng-class="chat.chatHistory.isEmpty() ? 'no-content' : ''">
 
     <div class="chat-loading" ng-if="loadingChat" onto-loader-new size="40"></div>
 
-    <div class="messages-hint text-muted" ng-if="!loadingChat && chat.chatHistory.isEmpty() && !askingChatItem">
-        {{ 'ttyg.chat_panel.hint' | translate }}
-    </div>
-
-    <div class="chat-details" ng-class="{'hidden': !chat}" ng-if="!loadingChat">
+    <div class="chat-details" ng-class="{'hidden': !chat}" ng-if="!loadingChat && !chat.chatHistory.isEmpty()">
```

Comment:
The ng-if condition has to be `ng-if="!loadingChat && !chat.chatHistory.isEmpty() || askingChatItem` to display the first question and loader immediately after clicking the ask button.